.vale/fixtures/RedHat/DoNotUseTerms/testinvalid.adoc

```diff
@@ -0,0 +1,29 @@
+congratulations
+daughterboard
+destroy
+dev
+domestic
+foo
+foreign
+fubar
+future-proof
+geo
+geography
+latest
```

Comment:
Hi, I was wondering, does adding these terms mean that Vale will fail on them? This is the only one that I question, because I think "install the latest version of the software" might be a fairly common directive and one where context matters. I'm unclear if failing Vale would prevent the build from happening. (ACS has some exceptions to Vale so we have it set up to allow the build, but I wasn't sure about OCP.)

Reply:
Agree I'll remove this thanks